public/ge0.html

```diff
@@ -20,97 +20,207 @@
+      margin-top: 2px;
+      margin-bottom: 2px;
+      display: inline-block;
+    }
+    
+    .linkcolour {
+      border-radius: 25px;
       padding: 5px;
+      padding-left: 10px;
+      padding-right: 10px;
       text-decoration: none;
+      color: #449f33;
+      vertical-align: middle;
       display: inline-block;
-      border: 1px solid #00AA00;
     }
-
-    .link:hover {
-      background-color: #00AA00;
-      color: white;
-      cursor: grab;
+    
+    .linkcolour:hover {
+      background-color: #449f33;
+      color: #FFF;
+      cursor: pointer;
     }
-
-    a,
-    a:link,
-    a:visited {}
-
-    a:hover,
-    a:active {}
-
-    #name {
-      font-size: x-large;
-      font-weight: bold;
-      margin: 0 5px 0 5px;
+    
+    .linkcolour:active {
+      background-color: #24541b;
     }
 
-    #header {
-      color: white;
-      background-color: #00AA00AA;
+    #container {
+      background-color: #FFF;
       text-align: center;
-      position: fixed;
-      width: 100%;
-      z-index: 1000;
       padding: 10px;
+      box-shadow: 0 3px 10px 0 #00000030;
+      display: flex;
+      flex-flow: row wrap;
+      align-items: center;
+      gap: 10px;
+      justify-content: space-around;
+      overflow-wrap: break-word;
+    }
+    
+    #header {
+      z-index: 1000;
+      width: 100%;
+      position: fixed;      
+    }
+    
+    #logo {
+      vertical-align: middle; 
+      height: 2em;
+      flex-flow: row wrap;
+      align-items: center;
+      gap: 10px;
+      display: inline-flex;
+      justify-content: space-between;
+      overflow-wrap: break-word;
+    }
+    
+    #name {
+      flex: 1;
+      font-size: x-large;
     }
 
     #map {
       height: 100%;
       width: 100%;
     }
+    
+    #notif {
+      z-index: 99999999;
+      border-radius: 25px;
+      padding: 5px;
+      padding-left: 10px;
+      padding-right: 10px;
+      overflow-wrap: break-word;
+      box-shadow: 0 3px 10px 0 #00000030;
+      visibility: hidden; 
+      background-color: #FFF;
+      color: #449f33;
+      position:absolute;
+      transform: translateX(-50%);
+      left:50%;
+      top:10%;
+    }    
+          
+    #notif.show {
+      visibility: visible;
+      animation: fadein 0.5s, fadeout 0.5s 1s;
+    }
+    
+    @keyframes fadein {
+      from {opacity: 0;}
+      to {opacity: 1;}
+    }
+
+    @keyframes fadeout {
+      from {opacity: 1;}
+      to {opacity: 0;}
+    }
   </style>
   <script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js" integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA==" crossorigin=""></script>
   <title>${title}</title>
 </head>
 
+<script>
+function openStore() {
+  window.location='/get.html';
+}
+
+function triggerApp() {
+  window.location='om:/${path}';
+  setTimeout(openStore, 700);
+}
+
+function copyCoords() {
+  var el = document.getElementById("notif");
+  el.className = "show";
```

Comment:
nit: consistently use single quotes in JS, and double quotes in HTML.

Reply:
fixed 👍